supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -149,128 +143,85 @@ Use "Memory Overcommitment Manager (MOM)" for the first instance in a section, a
-*Description*: Red Hat Enterprise Virtualization Manager Reports is available as an optional component. It produces reports that can be built and accessed through a web user interface, and then rendered to screen, printed, or exported to a variety of formats.
-
-This component was removed from Red Hat Virtualization (4.x), but still exists in Red Hat Enterprise Virtualization (3.x).
+[[red-hat-enterprise-linux-host]]
+==== image:images/yes.png[yes] Red Hat Enterprise Linux host (noun)
+*Description*: Red Hat Enterprise Linux servers subscribed to the appropriate entitlements can be used as hosts in Red Hat Virtualization.
 
-Spell out in full for the first instance in a section, and use "Reports" (always with a capital R) for subsequent instances.
+Always spell out in full. Do not capitalize "host".
 
 *Use it*: yes
 
-*Incorrect forms*: RHEVM Reports
+*Incorrect forms*: RHEL host, RHEL-H
 
-*See also*:
+*See also*: xref:host-rhv[host]
 
 [discrete]
 [[red-hat-virtualization]]
 ==== image:images/yes.png[yes] Red Hat Virtualization (noun)
 *Description*: Red Hat Virtualization is an enterprise-grade server and desktop virtualization platform built on Red Hat Enterprise Linux.
 
-Use "Red Hat Virtualization" for version 4.x. Always spell out in full, except as part of "RHVH" or when repetition in a single paragraph hampers readability.
+Use "Red Hat Virtualization". Always spell out in full, except as part of "RHVH" or when repetition in a single paragraph hampers readability.
 
 *Use it*: yes
 
 *Incorrect forms*: RHV
 
-*See also*: xref:red-hat-enterprise-virtualization[Red Hat Enterprise Virtualization], xref:red-hat-virtualization-host[Red Hat Virtualization Host]
+*See also*: xref:red-hat-virtualization-host[Red Hat Virtualization Host]
 
 [discrete]
 [[red-hat-virtualization-host]]
 ==== image:images/yes.png[yes] Red Hat Virtualization Host (noun)
-*Description*: Red Hat Virtualization Host is one of the types of host in Red Hat Virtualization (4.x). It is a minimal operating system based on Red Hat Enterprise Linux, is distributed as an ISO file from the Customer Portal, and contains only the packages required for the machine to act as a host. It is an improved version of Red Hat Enterprise Virtualization Hypervisor.
+*Description*: Red Hat Virtualization Host is the host in Red Hat Virtualization. It is a minimal operating system based on Red Hat Enterprise Linux, is distributed as an ISO file from the Customer Portal, and contains only the packages required for the machine to act as a host.
 
-Use "Red Hat Virtualization Host (RHVH)" for the first instance in a section. "RHVH" can be used in subsequent instances. Do not use "the Host" with a capital H. Do not use in Red Hat Enterprise Virtualization 3.x.
+Use "Red Hat Virtualization Host (RHVH)" for the first instance in a section. "RHVH" can be used in subsequent instances. Do not use "the Host" with a capital H.
```

Comment:
Maybe something like do not use "the Host" or capitalize the term "host" when it is not used with the full product name.. or something like that. WDYT?

Reply:
On this line it'd be easy to make this passive sentence active:

`You can use "RHVH" in subsequent instances.`